autogen/configureModel.m

```diff
@@ -76,6 +77,18 @@
 mp.Ts_sim = 1/(mp.Fs * mp.nChannels);         % System clock period
 mp.rate_change = mp.Fs_system/mp.Fs;   % how much faster the system clock is to the sample clock
 
+regSize = size(mp.register);
+nRegs = regSize(1);
+tempRegs = containers.Map();
+for idx=1:nRegs
+    tempRegs(mp.register{idx}.name) = mp.register{idx};
+end
+
+
+mp.getReg = tempRegs;
```

Comment:
Kind of clever, making a `map` look like a getter method.

Reply:
Yeah, I apparently should have been reviewing our older issues better because I did not know about #6 but I have been wanting an option to get registers by name for a while now. Amusingly part of the reason it is `getReg` is I initially wrote a function but because of scope issues it wasn't working that well and I still had nothing better than `getReg` for a convenient alternative interface the default `mp.register`

Reply:
In your defense, I closed #6 quite a while ago, probably due to lack of activity or something.